product_docs/docs/pgd/5/index.mdx

```diff
@@ -48,6 +48,24 @@ navigation:
 
 EDB Postgres Distributed (PGD) provides multi-master replication and data distribution with advanced conflict management, data-loss protection, and [throughput up to 5X faster than native logical replication](https://www.enterprisedb.com/blog/performance-improvements-edb-postgres-distributed). It enables distributed Postgres clusters with high availability up to five 9s.
 
+
+<div class="container mt-4 mb-4 border border-primary">
+<div class="row bg-light">
+<div class="col-12 text-center px-3 py-3">
+Read about why PostgreSQL is better when it’s distributed with EDB Postgres Distributed in <a href="https://www.enterprisedb.com/distributed-postgresql-always-on-database-availability">Distributed PostgreSQL:The Key to Always On Database Availability</a>
```

Comment:
This sentence has a lot of "distributed" in it. Maybe we can change one instance to "leverages", "uses", "employes" or something like that? For example: 
```suggestion
Read about why PostgreSQL is better when it leverages EDB Postgres Distributed in <a href="https://www.enterprisedb.com/distributed-postgresql-always-on-database-availability">Distributed PostgreSQL: The Key to Always On Database Availability</a>
```

Reply:
The content being pointed to is about distributed postgresql. Hence the deliberate repetition in the standout text, to associate Distributed PostgreSQL with EDB Postgres Distributed.